sound/soc/sof/ipc.c

```diff
@@ -243,26 +222,16 @@ static int ipc_tx_next_msg(struct snd_sof_ipc *ipc)
 	spin_lock_irq(&sdev->ipc_lock);
 
 	/* send first message in TX list */
-	msg = list_first_entry(&ipc->tx_list, struct snd_sof_ipc_msg, list);
+	msg = &ipc->msg;
```

Comment:
@lyakh do we need the variable msg.can we just use &ipc->msg directly?

Reply:
I'll remove this function altogether.